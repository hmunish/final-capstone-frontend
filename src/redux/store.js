import { configureStore } from '@reduxjs/toolkit';
import loginReducer from './login/loginSlice';
import signupReducer from './signup/signupSlice';

const store = configureStore({
  reducer: {
    login: loginReducer,
    signup: signupReducer,
  },
});

export default store;
