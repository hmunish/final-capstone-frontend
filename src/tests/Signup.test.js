import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import Signup from '../components/signup/signup';
import store from '../redux/store';

test('Car Details components Match', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Provider store={store}>
        <Signup />
      </Provider>
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
