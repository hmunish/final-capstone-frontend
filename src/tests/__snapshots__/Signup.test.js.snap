// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Car Details components Match 1`] = `
<section
  className="signup"
>
  <form
    onSubmit={[Function]}
  >
    <input
      className="text-input"
      name="username"
      placeholder="Username"
      type="text"
    />
    <input
      className="form-submit"
      type="submit"
      value="Sign Up"
    />
  </form>
  <a
    className="form-switch-link"
    href="/signin"
    onClick={[Function]}
  >
    Already a user? Sign In
  </a>
</section>
`;
